_data/projects/to-Chat.yml

```diff
@@ -0,0 +1,14 @@
+name: to-Chat (2C)
+desc: A chat box that lets you create group sessions for conversation, without the hassle of logging in.
+site: https://github.com/SamarthRanjanRay/2C
+tags:
+- *nodejs*
+- *js*
+- *chat*
+- *chatbox*
+- *group*
+- *conversation*
+- *room*
+upforgrabs:
+  name: Improve the UI
+  link: https://github.com/SamarthRanjanRay/2C/issues/1
```

Comment:
We like projects listed on this site to have a curated set of issues that contributors can dive into. Pointing to a single issue that lacks details isn't a great way to get started, and what I'd like to see is:

 - this single issue converted into a number of tasks
 - each of these tasks is assigned a label like `help wanted` or `up-for-grabs` so it's easy to filter

Once that's setup, the `name` attribute on the previous line will be the label used for these issues, and the `link` will point to the URL like `https://github.com/SamarthRanjanRay/2C/labels/help%20wanted`.

I'm happy to help guide you through this process, if you have any other questions.